front/app/components/VoteInputs/multiple/AssignMultipleVotesInput/index.tsx

```diff
@@ -141,17 +144,26 @@ const AssignMultipleVotesInput = ({
   const maxVotesPerIdeaReached = votes === voting_max_votes_per_idea;
   const maxVotes = voting_max_total ?? 0;
 
-  const minusButtonDisabledMessage = getMinusButtonDisabledMessage(
-    basketSubmitted,
-    onIdeaPage
+  const action =
+    phase.attributes.voting_method === 'budgeting' ? 'budgeting' : 'voting';
```

Comment:
It looks like this line is called every time we call `getPermissionsDisabledMessage`. Could we move it into `getPermissionsDisabledMessage`? And pass `phase` or `voting_method` instead of `action`?

Reply:
This is only called for the voting method - did originally have it in the method but I put it outside so when I move other methods / actions to use this method it's more generic